{"name": "Login with valid phone number and password -- @1.1 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=116.0.5845.141); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF7862352A2+57122]\n\t(No symbol) [0x00007FF7861AEA92]\n\t(No symbol) [0x00007FF78607E3AB]\n\t(No symbol) [0x00007FF7860B7D3E]\n\t(No symbol) [0x00007FF7860B7E2C]\n\t(No symbol) [0x00007FF7860F0B67]\n\t(No symbol) [0x00007FF7860D701F]\n\t(No symbol) [0x00007FF7860EEB82]\n\t(No symbol) [0x00007FF7860D6DB3]\n\t(No symbol) [0x00007FF7860AD2B1]\n\t(No symbol) [0x00007FF7860AE494]\n\tGetHandleVerifier [0x00007FF7864DEF82+2849794]\n\tGetHandleVerifier [0x00007FF786531D24+3189156]\n\tGetHandleVerifier [0x00007FF78652ACAF+3160367]\n\tGetHandleVerifier [0x00007FF7862C6D06+653702]\n\t(No symbol) [0x00007FF7861BA208]\n\t(No symbol) [0x00007FF7861B62C4]\n\t(No symbol) [0x00007FF7861B63F6]\n\t(No symbol) [0x00007FF7861A67A3]\n\tBaseThreadInitThunk [0x00007FFE43FE7614+20]\n\tRtlUserThreadStart [0x00007FFE45E826B1+33]\n\n", "trace": "  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login.py\", line 9, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\home_page.py\", line 19, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 32, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 46, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I navigated to Login page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=116.0.5845.141); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF7862352A2+57122]\n\t(No symbol) [0x00007FF7861AEA92]\n\t(No symbol) [0x00007FF78607E3AB]\n\t(No symbol) [0x00007FF7860B7D3E]\n\t(No symbol) [0x00007FF7860B7E2C]\n\t(No symbol) [0x00007FF7860F0B67]\n\t(No symbol) [0x00007FF7860D701F]\n\t(No symbol) [0x00007FF7860EEB82]\n\t(No symbol) [0x00007FF7860D6DB3]\n\t(No symbol) [0x00007FF7860AD2B1]\n\t(No symbol) [0x00007FF7860AE494]\n\tGetHandleVerifier [0x00007FF7864DEF82+2849794]\n\tGetHandleVerifier [0x00007FF786531D24+3189156]\n\tGetHandleVerifier [0x00007FF78652ACAF+3160367]\n\tGetHandleVerifier [0x00007FF7862C6D06+653702]\n\t(No symbol) [0x00007FF7861BA208]\n\t(No symbol) [0x00007FF7861B62C4]\n\t(No symbol) [0x00007FF7861B63F6]\n\t(No symbol) [0x00007FF7861A67A3]\n\tBaseThreadInitThunk [0x00007FFE43FE7614+20]\n\tRtlUserThreadStart [0x00007FFE45E826B1+33]\n\n", "trace": "  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login.py\", line 9, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\home_page.py\", line 19, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 32, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\POC-\\features\\pages\\BasePage.py\", line 46, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "attachments": [{"name": "screenshot", "source": "e346f499-58cb-41e2-8c88-40c5ffef5e4b-attachment.png", "type": "image/png"}], "start": 1693570213445, "stop": 1693570213760}, {"name": "When I enter valid phone number as \"8105000676\" and valid password as \"Loki@1234\" into the fields", "status": "skipped", "start": 1693570216937, "stop": 1693570216937}, {"name": "And I click on Login button", "status": "skipped", "start": 1693570216937, "stop": 1693570216937}, {"name": "Then I should get logged in", "status": "skipped", "start": 1693570216937, "stop": 1693570216937}], "parameters": [{"name": "number", "value": "8105000676"}, {"name": "password", "value": "Loki@1234"}], "start": 1693570178163, "stop": 1693570216937, "uuid": "4cbb3c86-ff36-4356-9da0-4339a4a3f19d", "historyId": "aa0a6f1a55c37fbffdea15f4e9a87916", "fullName": "Login Functionality: Login with valid phone number and password", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "login"}, {"name": "feature", "value": "Login Functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}